src/merkle/path_set.rs

```diff
@@ -82,7 +83,7 @@ impl MerklePathSet {
     /// # Errors
     /// * The specified index is not valid for the depth of the structure.
     /// * Leaf with the requested path does not exist in the set.
-    pub fn get_leaf(&self, index: u64) -> Result<Word, MerkleError> {
+    pub fn get_leaf(&self, index: u64) -> Result<RpoDigest, MerkleError> {
```

Comment:
I'm thinking we should keep the interface here as it was - i.e., the return type should be `Result<Word, MerkleError>`.